packages/backend/src/tests/tigerbeetle.ts

```diff
@@ -15,7 +15,7 @@ export async function startTigerbeetleContainer(clusterId?: number): Promise<{
   const tigerbeetleClusterId = clusterId || Config.tigerbeetleClusterId
   const { name: tigerbeetleDir } = tmp.dirSync({ unsafeCleanup: true })
   const tigerbeetleFile = `cluster_${tigerbeetleClusterId}_replica_0_test.tigerbeetle`
-  const tigerbeetleContainerVersion = 'ghcr.io/tigerbeetle/tigerbeetle:0.14.176'
+  const tigerbeetleContainerVersion = 'ghcr.io/tigerbeetle/tigerbeetle:0.15.3'
```

Comment:
Is there a way to pull this from somewhere such that we don't have to go into this file every time we want to upgrade?

Reply:
Not a straightforward way, but the way would be to have a custom github action that would:
1. Look at the `package.json` (configured) `dependencies` version (The renovate bot would be responsible to bump the `package.json`)
2. Have a set of "write" files/lines that would be inspected and verified whether the version matches, if the version does not match, we have a PR created with the matching version

> The above is what I found yesterday, I am still thinking of a simpler solution, perhaps a GitHook, which would mean it would not be tied to GitHub Actions. 🤞

Reply:
@sabineschaller I have created a separate story for having an auto-update mechanism:
https://github.com/interledger/rafiki/issues/2661
Shall we merge this PR in the meantime (If you are happy with the changes)? 

I am also reviewing the current TB code in Rafiki.